p5js/01_ImageNet_Camera/index.html

```diff
@@ -2,19 +2,19 @@
 
 <head>
   <meta charset="UTF-8">
-  <title>Imagenet Webcam example</title>
+  <title>Video Camera Classification using p5.js</title>
 
   <script src="../libraries/p5.min.js"></script>
```

Comment:
I wonder if we should reference p5 via a CDN? I think it's confusing for beginners sometimes when libraries are not part of the sketch directory itself.

Reply:
What about a CDN for `ml5.js` also? We could write something in the README about CDN's and how to run these examples offline (if they can be run offline). That would be good to document in general in terms of examples that connect to the cloud. I'll add a github issue.

Reply:
Since we are using npm to publish the package, we can use this CDN:
https://unpkg.com/ml5